src/NuGet.Services.EndToEnd/Support/PackageProperties.cs

```diff
@@ -1,27 +1,38 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using NuGet.Packaging;
 using System.Collections.Generic;
 
 namespace NuGet.Services.EndToEnd.Support
 {
     public class PackageProperties
     {
-        public bool IsSymbolsPackage { get; }
+        public PackageType Type { get; }
 
         public HashSet<string> IndexedFiles { get; }
 
-        public PackageProperties()
+        public LicenseMetadata LicenseMetadata { get; }
+
+        public PackageProperties() { }
```

Comment:
>public [](start = 8, length = 6)

Why have a default ctor? Seems like `PackageType` should be required.

Reply:
Yes, That's true.

Reply:
Can we remove it?

Reply:
Delete it now.